src/e2e/java/teammates/e2e/pageobjects/AdminHomePage.java

```diff
@@ -69,13 +66,17 @@ public void queueInstructorForAdding(String instructorDetails) {
         click(submitButtonDetailsSingleLineForm);
     }
 
-    public void addAllInstructors() {
-        click(addAllInstructorsButton);
-        waitForElementToBeClickable(addAllInstructorsButton);
+    public void clickApproveAccountRequestButton(String name, String email, String institute) {
+        WebElement accountRequestRow = getAccountRequestRow(name, email, institute);
+        waitForElementPresence(By.cssSelector("[id^='approve-account-request-']"));
+        WebElement approveButton = accountRequestRow.findElement(By.cssSelector("[id^='approve-account-request-']"));
+        waitForElementToBeClickable(approveButton);
+        approveButton.click();
+        waitForPageToLoad();
     }
 
-    public String getMessageForInstructor(int i) {
-        By by = By.id("message-instructor-" + i);
+    public String getToastTextContent() {
```

Comment:
There should be a method (i think its named `verifyStatusMessage`) in the superclass `AppPage` to verify the toast contents. I'd suggest using that method as it has a retry mechanism

Reply:
Switched over to verifyStatusMessage, thanks.